db/schema.rb

```diff
@@ -40,7 +40,9 @@
     t.datetime "updated_at", null: false
     t.bigint "like_id"
     t.string "image"
+    t.bigint "user_id"
     t.index ["like_id"], name: "index_posts_on_like_id"
+    t.index ["user_id"], name: "index_posts_on_user_id"
   end
```

Comment:
Are both t.bigint "user_id" and t.index ["user_id"] needed? I don't know the answer, but feels like they do the same job.